scenario/generateodag.go

```diff
@@ -372,12 +379,12 @@ func GetOdagSelectionBindings(host string, odagLinkId string, sessionState *sess
 	return nil, errors.New("failed to find any bindings for the ODAG link") // no bueno
 }
 
-// createFieldListboxAsync creates a listbox session object for specified field
-func createFieldListboxAsync(sessionState *session.State, actionState *action.State, doc *enigma.Doc, field string) (*senseobjects.ListBox, error) {
-	var obj *senseobjects.ListBox
+// createOdagToolbarNavpointAsync creates a listbox session object for specified field
+func createOdagToolbarNavpointAsync(sessionState *session.State, actionState *action.State, doc *enigma.Doc, field string) (*senseobjects.OdagToolbarNavpoint, error) {
+	var obj *senseobjects.OdagToolbarNavpoint
 	err := sessionState.SendRequest(actionState, func(ctx context.Context) error {
 		var err error
-		obj, err = senseobjects.CreateListBoxObject(ctx, doc, field)
+		obj, err = senseobjects.CreateOdagToolbarNavpoint(ctx, doc, field)
```

Comment:
Is a new session object created for every ODAG request in client? E.g. considering this scenario
1. selects x,y
2. Create ODAG
3. select v,w
4. Create ODAG
Should 2 session objects be created or is the object in (2) updated through normal update mechanism and updated during (3)?

Reply:
They are always created (and instantly destroyed after the request has been sent). So it follows the client flow.

Reply:
ah I see when looking in full code, didn't show in diff, makes sense then